tony-core/src/main/java/com/linkedin/tony/TaskExecutor.java

```diff
@@ -303,30 +304,7 @@ public HdfsAvroFileSplitReader getHdfsAvroFileSplitReader(List<String> readPaths
     return new HdfsAvroFileSplitReader(hdfsConf, readPaths, this.taskIndex,
         this.numTasks, useRandomShuffle);
   }
-
-
-  //endregion
-
-  //region Testing
-
-  private void hangIfTesting() {
```

Comment:
is this no longer needed?

Reply:
It's no longer needed. I deleted the test that used it in https://github.com/linkedin/TonY/pull/69 but forgot to delete this method.